src/app/models/board.model.ts

```diff
@@ -73,21 +79,29 @@ export class BoardMapper implements Mapper<BoardModel, BoardModelUI> {
         fromPath: ['attributes', 'context'],
         toPath: ['context']
     }, {
-        fromPath: ['included'],
+        fromPath: ['relationships', 'columns', 'data'],
         toPath: ['columns'],
-        toFunction: (data) =>
-            Array.isArray(data) ? data.filter(d => d.type === 'boardcolumns') : []
+        toFunction: (data) => {
+            return data.map(col => {
```

Comment:
There is no check for the `data` being actually an array or not. If it sends null from the backend it'll fail.